### DISCUSSION TITLE
How to get started with ionic extensions in an new nx 17.2.2 workspace?

### DISCUSSION THREAD

**Discussion by vicatcu** [nxext/nx-extensions#1057] [Q&A]
I'm totally lost on how to get started from scratch making a new Ionic project in an nx workspace. All the tutorials online (including Ionic's own blog) are woefully out of date. Can anyone help? My broader goal is to make an electron app with an ionic frontend and looking at https://github.com/bennymeg/nx-electron as a means to that end.

**Comment by rbalet** [comment]
@vicatcu Could you find something ? 

This is the latest place I could find any useful information: https://nxext.dev/docs/ionic-angular/generators.html
